### PR TITLE
Add alternate button frames and misc frames

### PR DESCRIPTION
**Pull request by PsyCommando**
## Description of changes
Extra stuff that didn't fit in #2607.
Depends on #2607.

Made an item frame, kit, and recipe for the door control button, the airlock button and blast door button. Previously you couldn't build button with those appearences. 

They behave exactly like the regular button. This is mostly cosmetic, and to get access to the default part preset.

## Changelog
:cl:
add: Added wall frame item, kit, and fabricator recipe for building door, airlock, and blast door control button. They behave just like the current button frame and kit, except they have the expected icon, and stock part presets
add: Added a kit for building an airlock sensor.
/:cl:

### CODE REVIEW

File: code/game/machinery/alarm.dm [hunk 1]
```diff
@@ -58,8 +58,7 @@
 
 	base_type = /obj/machinery/alarm
 	frame_type = /obj/item/frame/air_alarm
-	stat_immune = 0
-	uncreated_component_parts = null
+	uncreated_component_parts = list(/obj/item/stock_parts/power/apc = 1)
```
Comment (MistakeNot4892) [inline code/game/machinery/alarm.dm]: What does this have to do with button frames?

File: code/game/machinery/atmoalter/meter.dm [hunk 2]
```diff
@@ -9,7 +9,7 @@
 	idle_power_usage = 15
 
 	uncreated_component_parts = list(
-		/obj/item/stock_parts/power/apc/buildable
```
Comment (MistakeNot4892) [inline code/game/machinery/atmoalter/meter.dm]: As above.

File: code/game/machinery/alarm.dm (hunk 1 above)
Comment (PsyCommando) [reply code/game/machinery/alarm.dm]: That was because the branch needed a rebase since it depended on something that has been merged #2607.

File: code/game/machinery/atmoalter/meter.dm (hunk 2 above)
Comment (PsyCommando) [reply code/game/machinery/atmoalter/meter.dm]: Replied above.

**Review by afterthought2 — APPROVED** [review]
